PowerEditor/src/WinControls/FunctionList/functionListPanel.cpp

```diff
@@ -557,6 +557,7 @@ void FunctionListPanel::notified(LPNMHDR notification)
 			case NM_DBLCLK:
 			{
 				openSelection(treeView);
+				PostMessage(_hParent, WM_COMMAND, SCEN_SETFOCUS << 16, reinterpret_cast<LPARAM>((*_ppEditView)->getHSelf()));

```

Comment:
Curious...why left shift 16?

Reply:
Because `SCEN_SETFOCUS` is expected in the high word of `wParam` in the Big Switch. It's a Scintilla-defined non-standard `WM_COMMAND` message. Now, since you asked, I could have used 

	PostMessage(_hParent, NPPM_INTERNAL_SWITCHVIEWFROMHWND, 0, reinterpret_cast<LPARAM>((*_ppEditView)->getHSelf()));

too, it does absolutely the same and looks even better.

Reply:
Or use `MAKEWPARAM`-macro:
```cpp
PostMessage(_hParent, WM_COMMAND, MAKEWPARAM(SCEN_SETFOCUS, 0), reinterpret_cast<LPARAM>((*_ppEditView)->getHSelf()));
```

Reply:
That's, what you mean. It wouldn't have appeared to me in a dream to use `MAKEWPARAM`, but you are right. So, if I would change this one again, I would go for the `NPP_INTERNAL_SWITCHVIEWFROMHWND` message. But in the future, I will use the `MAKE*PARAM` macros for such things, if possible. Thank you for the hint.